SmoothScrolling/Client/TableView/TableView/UserCell.swift

```diff
@@ -25,16 +25,11 @@ class UserCell: UITableViewCell {
     
     fileprivate var downloadTask: URLSessionDataTask?
     
-    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
-        super.init(style: style, reuseIdentifier: reuseIdentifier)
-        
+    override func awakeFromNib() {
+        super.awakeFromNib()
         avatar.setRoundedImage(UserCell.defaultAvatar)
-        
```

Comment:
A further optimization would be to move `setOpaqueBackground` inside `awakeFromNib()`.